proto/job.proto

```diff
@@ -147,6 +147,9 @@ message RayJob {
   // Optional entrypointResources specifies the custom resources and quantities to reserve
   // for the entrypoint command.
   string entrypointResources = 20;
+  // Optional activeDeadlineSeconds is the duration in seconds that the RayJob may be active before
+  // KubeRay actively tries to terminate the RayJob; value must be positive integer
+  int32 activeDeadlineSeconds = 25;
```

Comment:
Could you please add a documentation on unset (aka, default value 0)'s behavior?